test/react_test_utils_test.dart

```diff
@@ -102,16 +102,29 @@ void main() {
       group('dragStart', () => testEvent(Simulate.dragStart, 'dragStart'));
       group('drop', () => testEvent(Simulate.drop, 'drop'));
       group('focus', () => testEvent(Simulate.focus, 'focus'));
+      group('gotPointerCapture',
```

Comment:
```suggestion
      group('gotPointerCapture', () => testEvent(Simulate.input, 'gotPointerCapture'));
```

Reply:
Same thing as above, just the 120 character limit